components/raftstore/src/store/snap.rs

```diff
@@ -1168,6 +1171,16 @@ impl SnapManager {
         Ok(())
     }
 
+    pub fn update_store_status(&self, status: StoreStatus) {
+        let new_limit = match status {
+            StoreStatus::Normal => self.max_write_speed as f64,
+            StoreStatus::OngoingOnline => f64::INFINITY,
```

Comment:
How to define the completion staus of the online?

Reply:
PD determines it based on expected region score and exact region score of the store. For example, if exact region score reaches 90% of the expected one, PD can think the store becomes normal.